### PR TITLE
More updates to address needed components

### PR DESCRIPTION
**Pull request by ctb**
(not yet ready for merge, review welcome)

### CODE REVIEW

**Comment by ctb** [comment]
Ready for review.

**Comment by ctb** [comment]
& merge.

File: proposal.md [hunk 1]
```diff
@@ -263,14 +266,24 @@ open-source communities as demonstrated by the interest in the
...
 publishers.
 
-Concretely we propose to: build a web application for the publication
+Concretely, we propose to: build a web application for the publication
 of dynamic and interactive scientific narratives, create tools and
 educational material to accelerate the adoption of existing solutions
 for reusable science, and finally to build a community around these
 ideas.
 
-This will be amazing!
```
Comment (betatim) [inline proposal.md]: wut?! It had a good run though.

**Comment by betatim** [comment]
LGTM.

**Pull request merged**